.vsts-ci/mac.yml

```diff
@@ -94,6 +94,10 @@ stages:
       vmImage: macOS-latest
     steps:
     - pwsh: |
+        # Remove old .NET SDKs
```

Comment:
This probably has to be done in the release build as well

Reply:
For some reason release build is fine.